frontend/components/SignUpForNotifications.js

```diff
@@ -47,7 +59,16 @@ class SignUpForNotifications extends React.Component {
       return;
     }
 
-    window.FB.XFBML.parse(this.facebookScopeRef);
+    // firefox has a strict no-trackers rule, which FB violates
+    // this is to make sure the site doesn't crash on firefox
+    if (window.FB) {
+      window.FB.XFBML.parse(this.facebookScopeRef);
+    } else {
+      this.setState({ // eslint-disable-line react/no-did-update-set-state
```

Comment:
Lets not use `react/no-did-update-set-state` - lets figure out the right way to do this!

Reply:
ping